content/api/vision.md

```diff
@@ -31,31 +31,35 @@ for marker in markers:
 ## Marker properties
 The markers that are contained within the array have some useful properties:
 
-  - `is_wall_marker()` - returns whether or not the marker is a wall marker.
-  - `is_token_marker()` - returns whether or not the marker is a token marker.
-  - `id()` - returns the id of the marker
-  - `pixel_centre()` - returns a [tuple](https://www.tutorialspoint.com/python/python_tuples.htm) containing the pixel coordinates of the centre of the marker.
-  - `polar()` - returns a PolarCoord object containing the position of the marker in the Polar coordinate system.
-  - `cartersian()` - returns a CartCoord object containing the position of the marker in the Cartesian coordinate system.
+  - `is_wall_marker` - returns whether or not the marker is a wall marker.
+  - `is_token_marker` - returns whether or not the marker is a token marker.
+  - `id` - returns the id of the marker
+  - `pixel_centre` - returns a [tuple](https://www.tutorialspoint.com/python/python_tuples.htm) containing the pixel coordinates of the centre of the marker.
+  - `polar` - returns a `PolarCoord` object containing the position of the marker in the Polar coordinate system.
+  - `cartesian` - returns a `CartCoord` object containing the position of the marker in the Cartesian coordinate system.
 
 ## Cartesian and Polar Coordinate Systems
-The markers' position can be represented using both the Polar and Cartesian coordiate system. The two objects `CartCoord` and `PolarCoord` hold the position information for both of these systems.
+The markers' position can be represented using both the Polar and Cartesian coordinate system. The two objects `CartCoord` and `PolarCoord` hold the position information for both of these systems.
 
 {{% notice info %}}
 [Cartesian coordinates](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) are ones that are defined using a unique set of values (typically x, y and z) to specify a point in space. [Polar coordinates](https://en.wikipedia.org/wiki/Polar_coordinate_system) are represented by a distance from a set point and an angle from a set direction.
 {{% /notice %}}
 
+{{% notice info %}}
+All distances in this section are measured in meters.
+{{% /notice %}}
+
 The `CartCoord` object has the following properties:
 
-  - `x()` - returns the x coordinate of the marker in Cartesian space, x is defined as being positive in the right direction.
-  - `y()` - returns the y coordinate of the marker in Cartesian space, y is defined as being positive in the up direction.
-  - `z()` - returns the z coordinate of the marker in Cartesian space, it is defined as the distance from the camera to the marker in **meters**
-  
+  - `x` - returns the x coordinate of the marker in Cartesian space, x is defined as being positive in the right direction.
```

Comment:
You can probably generalize this entire section. It has 3 properties (x, y and z), measured in meters, and the axis are relative to your robot (or its webcam anyway)